test/connection-test.ts

```diff
@@ -573,7 +540,6 @@ describe('Connection', function() {
 
   it('should try to reconnect on empty subscribe response on reconnect', function(done) {
```

Comment:
@intelliot would love some guidance here. Based on the old & new logic, an "empty subscribe response" would cause a connection failure, and not a second reconnect. But then this test seems to be testing that only a single reconnect happens, and `{returnEmptySubscribeRequest: 3}` doesn't seem to be doing anything (only `returnEmptySubscribeRequest: 1` seems to have an effect in mock-rippled).

I think this test is safe to remove, since we're already testing the reconnect logic and what happens when the server doesn't have validated ledgers.

Reply:
I would be fine with removing this test.